modules/parser/src/synchronizers/log-slice.js

```diff
@@ -114,7 +114,11 @@ export default class LogSlice {
     // get data if we don't already have that stream && it is not filtered.
     streamsByReverseTime.forEach(streams => {
       for (const streamName in streams) {
-        if (!this.streams[streamName] && this._includeStream(filter, streamName)) {
+        if (
+          this.streams[streamName] !== null && // Explicit no data entry
+          !this.streams[streamName] && // undefined means it has not been seen so keep looking for valid entry
```

Comment:
`!this.streams[streamName]` already covers the `null` case

Reply:
Actually the first explicit check for null is what makes using **null** work because if it is null we don't call **addStream**.  The 2 condition covers undefined or other falsy values, which covers the implicitly undefined entries in the stream arrays.

I hoped the comments made this clear by maybe I should expand.